protocols/identify/src/identify.rs

```diff
@@ -174,6 +175,11 @@ impl Identify {
         }
     }
 
+    /// Returns the `agent_version` of the node.
+    pub fn agent_version(&self) -> &str {
```

Comment:
Can you expand on why you need this function? After all the `agent_version` has been defined by the user already.

Reply:
Useful for debugging. Since it's an arbitrary string you can include it in log messages to identify the swarm they originated from. Peerids are less suited and since they are generated randomly also aren't the same across test runs

Reply:
Actually substrate uses it in that way too

Reply:
Right, but why offer a getter for a static value that the caller passed at construction time. Am I missing something?

Reply:
Why store multiple copies of the same string in different places for no reason? But sure it is possible to copy the string around which is what I've been doing so far.

Reply:
I guess I can live without